ansible/tasks/security/main.yml

```diff
@@ -11,7 +11,7 @@
 - name: "Per-user resource limits: Configure systemd resource accounting"
   blockinfile:
     path: /etc/systemd/system.conf
-    marker: "# {mark} ANSIBLE - Config for better resource accounting"
+    #marker: "# {mark} ANSIBLE - Config for better resource accounting"
```

Comment:
Why is this now commented out?

Reply:
Good question. I do not know, will revert that.